.gitignore

```diff
@@ -7,6 +7,9 @@ src/python/build/
 *.idea/
 *.vscode/
 
+# Intellij IDE
+src/ui/frontend.iml
```

Comment:
what is this file? Curious as I use IJ/webstorm too and haven't seen this file get generated.

Reply:
I have no idea where it came from but it appeared in this PR :o

Reply:
okay, no worries. It's just something in our .gitignore, so it should be okay I suppose.